torch/_jit_internal.py

```diff
@@ -704,7 +703,17 @@ def copy_torchscript_modifier(orig, new) -> None:
 # qualified_name => list[overload_functions]
 _overloaded_fns : Dict[str, List[Callable]] = {}  # noqa: T484
 
+def _check_overload_body(func):
+    parsed_def = parse_def(func)
+    body = parsed_def.ast.body[0].body
+    if len(body) != 1 or not isinstance(body[0], ast.Pass):
+        msg = "Only `pass` statement can be the body of overload declaration:\n"
+        msg += '\n'.join(parsed_def.source.split("\n")[:3])
+        msg += " <- Expecting `pass` here! "
+        raise RuntimeError(msg)
```

Comment:
When user hits this error, chances are they do not know how to use `_overload` correctly. Maybe it is worthwhile to go extra mile by showing the correct way of using `_overload` here.